saya menambahkan code

jikgungde01 opened the pull request:
selamat malam pak ,saya gungde baru saja menambahkan code ,tolong di cek ya pak

jikgungde01 committed:
saya menambahkan code

selamat malam pak ,saya gungde baru saja menambahkan code ,tolong di cek ya pak

kyo21 reviewed (commented):
Ini kodenya lgsung selesai lho, klo sy approve, teman2 sekelompok yg lain akan dianggap tdk mengerjakan apa2. Yakin ya?

jikgungde01 commented:
baik pak saya ulang dulu pak tdi ada problem pak

kyo21 reviewed (changes requested):
Ok, revisi dulu

jikgungde01 committed:
menambahkan code void

selamat malam pak,saya gungde ,tadi saya sudah diskusi untuk membagi dan buat ulang code dan saya membuat voidnya
